### PR TITLE
ZH_TW docs api/structures/cookie add

### PR DESCRIPTION
**Pull request by excfergodst**
ZH_TW docs api/structures/cookie add

### CODE REVIEW

File: docs-translations/zh-TW/api/structures/cookie.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Cookie 物件
+
+* `name` 字串 - cookie 的名字.
+* `value` 字串 - cookie 的值.
+* `domain` 字串 - cookie 的域名.
+* `hostOnly` 字串 - cookie 是否為Host-Only.
```
Comment (chuanxd) [inline docs-translations/zh-TW/api/structures/cookie.md]: space 
`是否為 Host-Only.`

File: docs-translations/zh-TW/api/structures/cookie.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* `path` 字串 - cookie 的路徑.
+* `secure` 布林 - cookie 的網域是否安全 （https).
+* `httpOnly` 布林 - cookie 是否只能運行在 HTTP.
+* `session` 布林 - cookie 為 短期session 或者 長期cookie，若是長期cookie得包含一個截止日期.
```
Comment (chuanxd) [inline docs-translations/zh-TW/api/structures/cookie.md]: `cookie 為短期 session 或者長期 cookie，若是長期 cookie 得包含一個截止日期`

File: docs-translations/zh-TW/api/structures/task.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Task 物件
+
+* `program` 字串 - 程式執行的路徑，通常你應該指定`process.execPath`，此值為當前程式執行的路徑.
```
Comment (chuanxd) [inline docs-translations/zh-TW/api/structures/task.md]:
```
通常你應該指定 `process.execPath`，此值為當前程式執行的路徑
```

**Comment by chuanxd** [comment]
@excfergodst please using chinese period `。`

**Comment by zeke** [comment]
> chinese period 。

Interesting, I'd never heard of this.

> The Chinese full stop is a small circle that takes the space of one Chinese character.

**Comment by kevinsawicki** [comment]
Thanks for updating these 👍

**Pull request merged**